website/docs/r/scc_scope.html.markdown

```diff
@@ -0,0 +1,106 @@
+  * Constraints: The maximum length is `512` characters. The minimum length is `0` characters. The value must match regular expression `/^[a-zA-Z0-9_,'\\s\\-\\.]*$/`.
+* `environment` - (Optional, String) The scope environment. This value details what cloud provider the scope targets.
+  * Constraints: The maximum length is `128` characters. The minimum length is `0` characters. The value must match regular expression `/^[a-zA-Z0-9_,'\\s\\-\\.]*$/`.
+* `instance_id` - (Required, Forces new resource, String) The ID of the Security and Compliance Center instance.
+  * Constraints: The maximum length is `36` characters. The minimum length is `36` characters. The value must match regular expression `/^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$/`.
+* `name` - (Optional, String) The scope name.
+  * Constraints: The maximum length is `128` characters. The minimum length is `1` character. The value must match regular expression `/^[a-zA-Z0-9_,'\\s\\-\\.]*$/`.
+* `properties` - (Optional, List) The span for the scope to target.
+Nested schema for **properties**:
+    * `account_id` - (Optional, ForceNew, String) The ID of the IBM account ID.
+    * `enterprise_id` - (Optional, ForceNew, String) The ID of the IBM enterprise ID.
+    * `resource_group_id` - (Optional, ForceNew, String) The ID of the IBM resource group tied to an account
+    * `account_group_id` - (Optional, ForceNew, String) The ID of an account group tied to an enterprise
+    * `ibm_facts_api_instance_id` - (Optional, ForceNew, String) The ID of ibm_cloud_facts_api provider type instance
+      * Constraints: Using `ibm_facts_api_instance_id` is restricted to certain users. 
+
+* `exclusions` - (Optional, List) A list of scopes/targets to exclude from a scope.
+Nested schema for **exclusions**:
+    * `account_id` - (Optional, String) The account ID to exclude.
+    * `resource_group_id` - (Optional, String) The ID of the IBM resource group in an account to exclude
+    * `account_group_id` - (Optional, String) The ID of an account group in an enterprise.
+
+## Attribute Reference
+
+After your resource is created, you can read values from the listed arguments and the following attributes.
+
+* `id` - The unique identifier of the scc_scope.
+* `account_id` - (String) The ID of the account associated with the scope.
+  * Constraints: The maximum length is `32` characters. The minimum length is `32` characters. The value must match regular expression `/^[a-zA-Z0-9_\\-.]*$/`.
+* `attachment_count` - (Float) The number of attachments tied to the scope.
+* `created_by` - (String) The identifier of the account or service ID who created the scope.
+  * Constraints: The maximum length is `255` characters. The minimum length is `1` character. The value must match regular expression `/^[a-zA-Z0-9-\\.:,_\\s]*$/`.
+* `created_on` - (String) The date when the scope was created.
+* `scope_id` - (String) The ID of the scope.
```

Comment:
not right key should be id

Reply:
A terraform resource has a special usage of `id`. If `id` were to be used, it will fail.